public/stylesheets/style.css

```diff
@@ -185,4 +198,190 @@ img#start-image {
 
 .canvas-main {
     background-color: blue;
-}
\ No newline at end of file
+}
+
+/*
+====================
+3 dots nav
+====================
+*/
+nav {
+    position: fixed;
+    height: 100vh;
+    width: 18px;
+    margin-left: 80px;
+    z-index: 3;
+}
+
+nav .wrapper {
+    position: absolute;
+    left: 0;
+    top: 50%;
+    transform: translateY(-50%);
+    height: 128px;
+    width: 18px;
+}
+
+nav .wrapper .outer-circle {
+    width: 18px;
+    height: 18px;
+    border-radius: 50%;
+    border: 1px solid #fafaff;
+    cursor: pointer;
+    position: absolute;
+    left: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+nav .wrapper .outer-circle:nth-child(1) {
+    top: 0;
+}
+
+nav .wrapper .outer-circle:nth-child(2) {
+    top: 55px;
+}
+
+nav .wrapper .outer-circle:nth-child(3) {
+    top: 110px;
+}
+
+nav .wrapper .outer-circle .inner-circle {
+    width: 14px;
+    height: 14px;
+    border-radius: 50%;
+    background-color: #fafaff;
+    display: none;
+}
+
+nav .wrapper .outer-circle .inner-circle.active {
+    display: block;
+}
+
+/*
+====================
+SECTION 1
+====================
+*/
+
+header {
+    /* background-color: black; */
+    color: #fafaff;
+    position: relative;
+}
+
+header .wrapper {
+    position: absolute;
+    z-index: 2;
+    padding: 20px 40px;
+}
+
+header .wrapper a {
+    text-decoration: none;
+    color: #fafaff;
+    display: flex;
+    align-items: center;
+}
+
+header .wrapper .logo {
+    width: 60px;
+}
+
+header .wrapper .herder-title {
```

Comment:
Typo, herder-title→header-title

Reply:
Thank you!
Fixed in this commit https://github.com/eastend-street/quote-maker/pull/7/commits/59ffc6a397fcb92886deea5f345ea2054c9d9fd2